docs/tutorials/developer-journey/level-4/4.4-nns-governance.md

```diff
@@ -70,6 +70,80 @@ There are several reasons that all users and developers on the IC should be fami
 
 - Gaining a comprehensive understanding of the NNS and SNSs, allowing you to envision creating an SNS for your project: By interacting with the NNS and the SNSs for different applications on the IC, developers can be inspired to turn their application into an SNS as well. 
 
+## What is an SNS?
+
+A Service Nervous System (SNS) is an advanced form of a DAO. The underlying architecture of an SNS closely resembles the architecture of the NNS, but instead of being used to control the IC, an SNS is used to control a specific decentralized application that is running on the IC mainnet. Any dapp running on the mainnet can be handed over to an SNS, which results in the dapp being owned and controlled by a community. When an SNS takes control of a dapp, the dapp is then managed similarly to how the Internet Computer is managed by the NNS. When an SNS is launched, a new unique token is created for the SNS that is then used by the community for participation in the SNS's governance.
+
+The dapp's community can participate in the dapp's SNS through staking the SNS's native token in a neuron. Then, changes to the dapp are proposed to the community via an SNS proposal. SNS voting rights use the same format that voting on the NNS does; anyone can stake the SNS's native token into a neuron, and the neuron's voting power is proportional to the quantity of SNS tokens staked within the neuron, the length of their dissolve delay, and the amount of time since the neuron was created.
+
+You can learn more about SNSs in the following resources:
+
+* [SNS overview page](https://internetcomputer.org/sns).
+
+* [Service Nervous System (SNS) wiki page](https://wiki.internetcomputer.org/wiki/Service_Nervous_System_(SNS)).
```

Comment:
Should we rather link to the developer docs than the Wiki (I think the wiki might not be there in the long term and this is targeted at developers). 
This might be a good page for a dev to get an overview of all documentation relate to SNS : https://internetcomputer.org/docs/current/developer-docs/integrations/sns/